releases/2020.08.feat-01.yaml

```
@@ -0,0 +1,145 @@
+epositories:
+  qpOASES:
+    type: git
+    url: https://github.com/robotology-dependencies/qpOASES.git
+    version: v3.2.0.1
+  osqp:
+    type: git
+    url: https://github.com/oxfordcontrol/osqp.git
+    version: v0.6.0
+  ycm:
```

```suggestion
  YCM:
```
the lowercase ycm that you can find in other files was a bug.